buildSrc/src/main/java/com/uber/okbuck/core/util/windowsfs/WindowsFS.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2017-present Facebook, Inc.
```

Comment:
Can you please remove this copyright

Reply:
This class was copied from Buck repo: https://github.com/facebook/buck/blob/221c4c8dd31b301910d0a8abae121308004a30e3/src/com/facebook/buck/io/windowsfs/WindowsFS.java

Apache license requires to keep it while you still can use and modify this code without limitations. 
Are you ok if I stack it with MIT license? Or I can remove headers and place LICENSE file in buildSrc/src/main/java/com/uber/okbuck/core/util/windowsfs/.

Reply:
1. Keep the copyright header of WindowsFS.java
2. Add the following attribution to the bottom of okbuck LICENSE file in the root directory: 
> okbuck includes derived work from buck(https://github.com/facebook/buck/) under the Apache License 2.0: 
> 
> Copyright 2017-present Facebook, Inc.
>  
> Licensed under the Apache License, Version 2.0 (the "License"); you may
> not use this file except in compliance with the License. You may obtain
> a copy of the License at
>  
> http://www.apache.org/licenses/LICENSE-2.0
> 
> Unless required by applicable law or agreed to in writing, software
> istributed under the License is distributed on an "AS IS" BASIS, WITHOUT
> WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
> License for the specific language governing permissions and limitations
> under the License.
> 
> The derived work can be found in the files: buildSrc/src/main/java/com/uber/okbuck/core/util/windowsfs/WindowsFS.java
> 

3) update okbuck's copyright years 

> Copyright (c) 2016-2019 Uber Technologies, Inc.

Reply:
@KapJI can you make these changes?